src/gui/properties/speedplotview.cpp

```diff
@@ -137,56 +173,56 @@ void SpeedPlotView::paintEvent(QPaintEvent *)
 {
     QPainter painter(this->viewport());
 
-    QRect full_rect = this->viewport()->rect();
+    QRect fullRect = this->viewport()->rect();
     QRect rect = this->viewport()->rect();
-    QFontMetrics font_metrics = painter.fontMetrics();
+    QFontMetrics fontMetrics = painter.fontMetrics();
 
     rect.adjust(4, 4, 0, -4); // Add padding
 
-    double max_y = maxYValue();
+    int maxY = maxYValue();
 
-    rect.adjust(0, font_metrics.height(), 0, 0); // Add top padding for top speed text
+    rect.adjust(0, fontMetrics.height(), 0, 0); // Add top padding for top speed text
 
     // draw Y axis speed labels
-    QVector<QString> speed_labels(QVector<QString>() <<
-                                  Utils::Misc::friendlyUnit(max_y, true) <<
-                                  Utils::Misc::friendlyUnit(0.75 * max_y, true) <<
-                                  Utils::Misc::friendlyUnit(0.5 * max_y, true) <<
-                                  Utils::Misc::friendlyUnit(0.25 * max_y, true) <<
-                                  Utils::Misc::friendlyUnit(0, true));
-
-    int y_axe_width = 0;
-    for (int i = 0; i < speed_labels.size(); ++i) {
-        if (font_metrics.width(speed_labels[i]) > y_axe_width)
-            y_axe_width = font_metrics.width(speed_labels[i]);
+    QVector<QString> speedLabels(QVector<QString>() <<
+                                 Utils::Misc::friendlyUnit(maxY, true) <<
+                                 Utils::Misc::friendlyUnit(0.75 * maxY, true) <<
+                                 Utils::Misc::friendlyUnit(0.5 * maxY, true) <<
+                                 Utils::Misc::friendlyUnit(0.25 * maxY, true) <<
+                                 Utils::Misc::friendlyUnit(0, true));
+
+    int yAxeWidth = 0;
+    for (int i = 0; i < speedLabels.size(); ++i) {
+        if (fontMetrics.width(speedLabels[i]) > yAxeWidth)
+            yAxeWidth = fontMetrics.width(speedLabels[i]);
     }
 
-    for (int i = 0; i < speed_labels.size(); ++i) {
-        QRectF label_rect(rect.topLeft() + QPointF(-y_axe_width, i * 0.25 * rect.height() - font_metrics.height()),
-                          QSizeF(2 * y_axe_width, font_metrics.height()));
-        painter.drawText(label_rect, speed_labels[i], QTextOption((Qt::AlignRight) | (Qt::AlignTop)));
+    for (int i = 0; i < speedLabels.size(); ++i) {
+        QRectF label_rect(rect.topLeft() + QPointF(-yAxeWidth, i * 0.25 * rect.height() - fontMetrics.height()),
+                          QSizeF(2 * yAxeWidth, fontMetrics.height()));
+        painter.drawText(label_rect, speedLabels[i], QTextOption((Qt::AlignRight) | (Qt::AlignTop)));
```

Comment:
Why is it so hard?
```c++
QTextOption((Qt::AlignRight) | (Qt::AlignTop))
```
Because you can simply do this:
```c++
Qt::AlignRight | Qt::AlignTop
```